source/simulator/core.cc

```diff
@@ -168,75 +168,78 @@ namespace aspect
                      pcout,
                      TimerOutput::never,
                      TimerOutput::wall_times),
-    initial_topography_model(InitialTopographyModel::create_initial_topography_model<dim>(prm)),
-    geometry_model (GeometryModel::create_geometry_model<dim>(prm)),
-    // make sure the parameters object gets a chance to
-    // parse those parameters that depend on symbolic names
-    // for boundary components
-    post_geometry_model_creation_action (std::bind (&Parameters<dim>::parse_geometry_dependent_parameters,
-                                                    std::ref(parameters),
-                                                    std::ref(prm),
-                                                    std::cref(*geometry_model))),
-    material_model (MaterialModel::create_material_model<dim>(prm)),
-    gravity_model (GravityModel::create_gravity_model<dim>(prm)),
-    prescribed_stokes_solution (PrescribedStokesSolution::create_prescribed_stokes_solution<dim>(prm)),
-    adiabatic_conditions (AdiabaticConditions::create_adiabatic_conditions<dim>(prm)),
+    Timer walltime;
```

Comment:
you are in the middle of a class member initializer list (https://en.cppreference.com/w/cpp/language/constructor). You can not declare a new variable here. Move this into the class Simulator in simulator.h

Reply:
ah yes thanks